docs/releases/2.1.txt

```diff
@@ -222,7 +222,8 @@ supports PostgreSQL 9.4 and higher.
 Miscellaneous
 -------------
 
-* ...
+* :meth:`~django.db.models.query.QuerySet.reverse()` gave incorrect results if
```

Comment:
Since this is a bug in a new feature in 1.11, it qualifies for a backport to stable/1.11.x. Please put the reverse note in `docs/releases/1.11.7.txt`:
```
* Made ``QuerySet.reverse()`` affect ``nulls_first`` and ``nulls_last``
  (:ticket:`28722`).
```

Reply:
OK

Reply:
I can write:

```* Made ``OrderBy.reverse_ordering()`` affect ``nulls_first`` and ``nulls_last`` 
  (:ticket:`28722`).```

However, I suspect nobody is familiar with this API directly, they are using `QuerySet.reverse` instead, so I would prefer:

```* Fixed ``QuerySet.reverse()`` giving incorrect results when ``nulls_first`` or ``nulls_last`` was specified. (:ticket:`28722`).```

What do you think?